packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -418,6 +446,10 @@ class ThemeData {
   /// An icon theme that contrasts with the accent color.
   final IconThemeData accentIconTheme;
 
+  /// A slider theme that describes the colors and shapes used in
```

Comment:
Maybe: 

The colors and shapes used to render [Slider]. 

This is the default value of [SliderTheme.of].

Reply:
It's not really a default value: it's the value, however it got set.  But I like your idea, so I did something close.